apps/charterafrica/src/pages/index.page.js

```diff
@@ -89,6 +89,36 @@ export async function getStaticProps({ defaultLocale, locale, locales }) {
 
   const ecosystem = blocks.find((block) => block.slug === "ecosystem") || null;
 
+  const hero = blocks.find((block) => block.slug === "hero") || {};
+
+  const heroSlides = hero?.slides?.map((slide) => {
+    const { title, background, links, ...other } = slide;
+
+    const formattedLinks = links.map((link) => {
+      const { icon, ...rest } = link;
+      return {
+        ...rest,
+        icon: { src: icon.url },
+      };
+    });
+
+    return {
+      title: {
+        color: title.color,
+        content: title.content[0].children[0].text,
```

Comment:
Why? Isn't `RichText` component not working?

Reply:
@kilemensi I have [updated](https://github.com/CodeForAfrica/ui/blob/c50a7bd02e51c29004ef00628bef1c165998fb79/apps/charterafrica/src/components/Hero/Slide.js#L46-L68) it to `Richtext` and its working well except when it is html. It renders `html` as text instead of `html` Like below: 
![html](https://user-images.githubusercontent.com/43873157/213709139-4753a7f8-57cd-459c-a02d-43945a37683d.png)

Reply:
Where @kelvinkipruto . Just tried it locally and for 👇🏽 content
```json
"title": {
            "color": "#ffffff",
            "content": [
              {
                "children": [
                  {
                    "text": "Sometext "
                  },
                  {
                    "text": "Styling",
                    "bold": true,
                    "italic": true
                  }
                ]
              }
            ]
          },
          "subheading": {
            "color": "#000000",
            "content": "Subheading"
          },
```

This is how the slide was rendered.
![Screenshot from 2023-01-20 16-54-41](https://user-images.githubusercontent.com/1779590/213713999-983960ad-d51f-4985-94f1-0f597ad5ddae.png)

Reply:
And 👇🏽 is how it looks in the CMS @kelvinkipruto 
![Uploading Screenshot from 2023-01-20 16-58-10.png…]()

Reply:
ooh man!!! I was typing the full `html` in the cms instead of using the leaves😔😔. @kilemensi

Reply:
... 🤣 🤣 🤣 🤣 🤣 You need a break! TGIF!